tests/util/api/test_remote_backend.py

```diff
@@ -70,8 +72,13 @@ def test_create(configuration: MagicMock, mocked_settings: MagicMock, mocked_aut
 
 
 def test_run(
```

Comment:
This test was and is ridiculous. I suggest "fixing" this by making it unnecessary (QI2-1212).

Reply:
I agree